euphonic/spectra/base.py

```diff
@@ -1239,23 +1252,24 @@ def apply_kinematic_constraints(spectrum: Spectrum2D,
     try:
         (1 * spectrum.x_data.units).to('1/angstrom')
     except DimensionalityError as error:
-        raise ValueError(
-            'x_data needs to have wavevector units (i.e. 1/length)',
-            ) from error
+        msg = 'x_data needs to have wavevector units (i.e. 1/length)'
+        raise ValueError(msg) from error
     try:
         (1 * spectrum.y_data.units).to('eV', 'spectroscopy')
     except DimensionalityError as error:
-        raise ValueError(
-            'y_data needs to have energy (or wavenumber) units',
-            ) from error
+        msg = 'y_data needs to have energy (or wavenumber) units'
+        raise ValueError(msg) from error
 
     momentum2_to_energy = 0.5 * (ureg('hbar^2 / neutron_mass')
                                  .to('meV angstrom^2'))
 
     if (e_i is None) == (e_f is None):
-        raise ValueError('Exactly one of e_i and e_f should be set. '
+        msg = (
+            'Exactly one of e_i and e_f should be set. '
                          '(The other value will be derived from energy '
-                         'transfer).')
+                         'transfer).'
+        )
```

Comment:
Bad indent?

Reply:
Might as well dedent_and_fill this, I suspect the message will otherwise wrap poorly.